### DISCUSSION TITLE
please help me to tell the nice games

### DISCUSSION THREAD

**Discussion by Ryo-hack** [NAIST-SE/AttractingContributors#93] [Show and tell]
please tell me the nice games.

https://github.com/Ryo-hack/AttractingContributors

**Comment by sergiodl-naist** [comment]
Added FFVII Remake 
Kind of linear but has a great story
Please visit and try to ask a question or recommend me one vtuber on my fork!
https://github.com/sergiodl-naist/AttractingContributors-VTubers

**Comment by takara-f** [comment]
I added Valorant!
Please help me if you can.
https://github.com/F-TKR/AttractingContributors-game
